wagtail/images/models.py

```diff
@@ -29,6 +31,9 @@
 from wagtail.search.queryset import SearchableQuerySetMixin
 
 
+LOGGER = logging.getLogger("wagtail.images")
```

Comment:
Question: Should we add some explicit documentation about the existence of this logger, and how to enable / use it?